### PR TITLE
half of the slim files to html

### PR DESCRIPTION
**Pull request by jotaalvim**

### CODE REVIEW

File: app/views/activities/_activities_breadcrumb.html.erb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    .row
+      .col-sm-6
+        = link_to activities_path do
+          h4 Atividades
+      .col-sm-6.hidden-xs.text-right
+        ol.breadcrumb
+          li
+            = link_to 'Home', root_path
+          li.active Atividades
+-- >
```
Comment (miguelbrandao) [inline app/views/activities/_activities_breadcrumb.html.erb:13]: Remove this

File: app/views/activities/_form.html.erb [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+;
+; _slim_controls2 << ("</li>".freeze); end; _slim_controls2 << ("</div>".freeze); end; _slim_controls2 << ((form.label :name,
+class: "activity-form-label").to_s);
+; _slim_controls2 << ("<br".freeze); _slim_controls2 << (" />".freeze);
+; _slim_controls2 << ((form.text_field :name,
+class: "activity-form-field").to_s);
+;
+; _slim_controls2 << ((form.label :location,
+class: "activity-form-label").to_s);
+; _slim_controls2 << ("<br".freeze); _slim_controls2 << (" />".freeze);
```
Comment (miguelbrandao) [inline app/views/activities/_form.html.erb:139]: remove the slim controlls

**Review by miguelbrandao — CHANGES_REQUESTED** [review]

File: app/views/activities/_activities_breadcrumb.html.erb [hunk 3]
```diff
@@ -0,0 +1,20 @@
+ <div class="breadcrumb-wrap">
+  <div class="container">
+    <div class="row">
+      <div class="col-sm-6">
+        <% link_to activities_path do %>
+          <h4> Atividades </h4>
+        <% end %>
+
```
Comment (nelsonmestevao) [inline app/views/activities/_activities_breadcrumb.html.erb:8]:
```suggestion
```

File: app/views/activities/_form.html.erb [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+  <div class="submit-field">
+    <%= form.submit class: "btn btn-lg btn-update border-theme" %>
+    <% if @activity.id != nil %> 
+      <%= link_to 'Cancel', activity_path(@activity),
+        data: { confirm: 'You will lose all modifications. Are you sure?' },
+        class: "btn btn-lg border-theme" %>
+      <%= link_to 'Destroy', :activity, method: :delete,
+        data: { confirm: 'Are you sure you want to delete this activity?' },
+        class: "btn btn-lg btn-delete border-theme" %>
```
Comment (nelsonmestevao) [inline app/views/activities/_form.html.erb:116]:
```suggestion
        class: "btn btn-lg btn-delete border-theme" %>
     <% end %>
```

**Comment by ruioliveira02** [comment]
We are refactoring this in Elixir, so this PR is no longer useful. Thank you for contributing pray

**Pull request closed without merging**